mlflow/sagemaker/__init__.py

```diff
@@ -236,28 +291,24 @@ def _upload_s3(local_model_path, bucket, prefix):
                 Tagging={'TagSet': [{'Key': 'SageMaker', 'Value': 'true'}, ]}
             )
             eprint('tag response', response)
-            return '{}/{}/{}'.format(s3.meta.endpoint_url, bucket, key)
-
+            return os.path.join(s3.meta.endpoint_url, bucket, key)
```

Comment:
This is supposed to return url so maybe don't use os.path.join.

Reply:
Got it. Replaced it with slightly-tweaked slash concatenation. Wanted to make sure that this would work if the arguments contained trailing slashes.